{"name": "头部统计", "status": "failed", "statusDetails": {"message": "Failed: 数据库查询结果为空，无法进行对比", "trace": "<ExceptionInfo 数据库查询结果为空，无法进行对比 tblen=16>"}, "steps": [{"name": "1. 请求头部统计", "status": "passed", "start": 1749710406429, "stop": 1749710406504}, {"name": "2. 校验接口请求是否成功", "status": "passed", "start": 1749710406504, "stop": 1749710406509}, {"name": "3. 获取返回数据vpsActiveNum", "status": "passed", "start": 1749710406509, "stop": 1749710406513}, {"name": "4. 校验接口返回数据是否和数据库的数据一致", "status": "failed", "statusDetails": {"message": "Failed: 数据库查询结果为空，无法进行对比\n", "trace": "  File \"D:\\pycharm_test\\kuangjia8\\tests\\test_api.py\", line 73, in test_getstatdata\n    pytest.fail(\"数据库查询结果为空，无法进行对比\")\n  File \"D:\\pycharm_test\\.venv\\lib\\site-packages\\_pytest\\outcomes.py\", line 198, in fail\n    raise Failed(msg=reason, pytrace=pytrace)\n"}, "start": 1749710406513, "stop": 1749710406578}], "attachments": [{"name": "log", "source": "5948168a-ea7e-4c1a-9e47-d3890ceb108c-attachment.txt", "type": "text/plain"}], "start": 1749710406429, "stop": 1749710406578, "uuid": "6bf70e56-3d3d-47bc-8e62-1b15ef0c7340", "historyId": "131af02cd86bc00c70573cec7625bf92", "testCaseId": "131af02cd86bc00c70573cec7625bf92", "fullName": "tests.test_api#test_getstatdata", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "host", "value": "DESKTOP-4S9CU1E"}, {"name": "thread", "value": "28272-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}]}